node/pkg/websocketfetcher/providers/gopax/gopax.go

```diff
@@ -104,7 +104,11 @@ func (f *GopaxFetcher) customReadFunc(ctx context.Context, conn *websocket.Conn)
 
 	if strings.HasPrefix(rawResponse, "\"primus::ping::") {
 		log.Debug().Str("Player", "Gopax").Msg("received pong")
-		f.Ws.RawWrite(ctx, "\"primus::pong::"+rawResponse[15:])
+		rawWriteErr := f.Ws.RawWrite(ctx, "\"primus::pong::"+rawResponse[15:])
+		if rawWriteErr != nil {
+			log.Error().Str("Player", "Gopax").Err(rawWriteErr).Msg("error in gopax.customReadFunc, failed to write pong")
+			return nil, rawWriteErr
+		}
```

Comment:
I don't think this is not a relevant pr with the pr subject? 😅

if you updated this because of linter failure, I'd like to suggest to just use just
```
_ = f.Ws.RawWrite(ctx, "\"primus::pong::"+rawResponse[15:])
```
without error handling. 

It's not a critical part of this fetcher and even if it fails, it'll auto reconnect to the ws api. 

And it's the library function call which  is not expected to fail often.

Reply:
Not insisting though, will leave this up to you